packages/web-components/src/components/ic-select/readme.md

```diff
@@ -54,13 +54,6 @@ Type: `Promise<void>`
 
 
 
-## CSS Custom Properties
```

Comment:
The readme is doing that strange thing were it removes items. Just need to run the build one more time.